src/routes/views/components/charts/trendChart/trendChart.tsx

```diff
@@ -299,19 +300,20 @@ class TrendChartBase extends React.Component<TrendChartProps, State> {
 
   // Returns onMouseOver, onMouseOut, and onClick events for the interactive legend
   private getEvents() {
+    const { chartName } = this.props;
     const { hiddenSeries, series } = this.state;
 
     const result = getInteractiveLegendEvents({
       chartNames: getChartNames(series),
       isHidden: index => isSeriesHidden(hiddenSeries, index),
-      legendName: 'legend',
+      legendName: chartName ? chartName + '-legend' : 'legend',
```

Comment:
I've been testing this locally and the highlighting doesn't work as expected. Some legend items remain greyed out. 

Probably a bug with the chart's `getInteractiveLegendEvents` function. I suspect "legend" may be hard coded somewhere?

Looking at the PatternFly code now to see if I can quickly fix that

Reply:
Created a PR to fix the legend issue in PatternFly.
https://github.com/patternfly/patternfly-react/pull/8035